src/transaction_base.js

```diff
@@ -80,8 +80,9 @@ export class TransactionBase {
     );
 
     if (requiresContractSignatures) {
-      throw new Error(
-        'Soroban contract signatures are not supported in this version of the SDK.'
+      // eslint-disable-next-line no-console
```

Comment:
seems like `requiresContractSignatures` is not relevant for now, maybe just delete it and implement when concrete. this message may trigger more support to explain reasons.

Reply:
No more support issues than we already have, and support is imminent anyway, so.

Reply:
yeah agreed